doc/rst/users-guide/base/basicValues.rst

```diff
@@ -69,7 +69,7 @@ To represent a smaller integral value as a ``uint``, type conversions
 .. TODO: hyperlink coercions above once that text is written
    
 
-Floating Point Values
+Floating-Point Values
```

Comment:
This still makes a "floating-point" change

Reply:
There may be other instances as well

Reply:
Sure, I will have a look.